setup/tinyHIPPOstart

```diff
@@ -10,8 +10,6 @@
 opkg update
 
 # Make sure Python3, Pip, misc packages are installed
-opkg install python3
-opkg install python3-pip
```

Comment:
Where will is this going to be installed now? Also edit comment on line #4

Reply:
python3 and python3-pip are dependencies in the Makefile. So when the package is first installed on the router, opkg will check that the router has python3 and pip before installing tinyHIPPO.

So there is no need to make redundant calls here.